diagnostic.md

```diff
@@ -27,7 +27,9 @@ on the body element. Use that variable to increase font-size of `h1`
 elements by 30%.
 
 ```scss
-// your response here
+h1 {
+  font-size: $base-font-size * 0.3333;
```

Comment:
Correct but the math is wrong. We want the `h1` to be **increased** in size by 30% so it would need to be:
```scss
h1 {
   font-size: $base-font-size * 1.3;
}
```